### PR TITLE
*: bump 0.10.0

### PR DESCRIPTION
**Pull request by BusyJay**

### CODE REVIEW

File: benchmark/README.md [hunk 1]
```diff
@@ -26,10 +26,10 @@ $ cargo build -p benchmark --release
 
 ```
 $ cd ../grpc
-$ python2.7 tools/run_tests/run_performance_tests.py -l rust
+$ python3 tools/run_tests/run_performance_tests.py -l rust
 ```
 
-Checkout `python2.7 tools/run_tests/run_performance_tests.py --help` to see custom options.
+Checkout `python3 tools/run_tests/run_performance_tests.py --help` to see custom options.
```
Comment (hunterlxt) [inline benchmark/README.md:32]: The reason for the change?

File: proto/Cargo.toml [hunk 2]
```diff
@@ -18,11 +18,11 @@ protobuf-codec = ["grpcio/protobuf-codec", "protobuf-build/grpcio-protobuf-codec
 prost-codec = ["prost-derive", "prost-types", "bytes", "lazy_static", "grpcio/prost-codec", "prost", "protobuf-build/grpcio-prost-codec"]
 
 [dependencies]
-grpcio = { path = "..", features = ["boringssl"], version = "0.9.0", default-features = false }
+grpcio = { path = "..", features = ["boringssl"], version = "0.10.0", default-features = false }
```
Comment (hunterlxt) [inline proto/Cargo.toml:21]: It looks like all local libs can be used without specifying a version?

**Review by hunterlxt — APPROVED** [review]
rest LGTM

File: benchmark/README.md (hunk 1 above)
Comment (BusyJay) [reply benchmark/README.md:32]: grpc has upgraded to python3. See https://github.com/grpc/grpc/pull/27135.

File: proto/Cargo.toml (hunk 2 above)
Comment (BusyJay) [reply proto/Cargo.toml:21]: Unless they need to be published to crates.io.

Comment (hunterlxt) [reply proto/Cargo.toml:21]: got it, `tests-and-examples` dont need be published to crates.io so the version can be ignored.

**Pull request merged**